src/sinks/gcp/stackdriver_logs.rs

```diff
@@ -33,34 +33,63 @@ enum HealthcheckError {
     NotFound,
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone, Default)]
+/// Configuration for the `gcp_stackdriver_logs` sink.
+#[configurable_component(sink)]
+#[derive(Clone, Debug, Default)]
 #[serde(deny_unknown_fields)]
 pub struct StackdriverConfig {
     #[serde(skip, default = "default_endpoint")]
     endpoint: String,
 
     #[serde(flatten)]
     pub log_name: StackdriverLogName,
+
+    /// The log ID to which to publish logs.
+    ///
+    /// This is a name you create to identify this log stream.
+    #[configurable(metadata(templateable))]
     pub log_id: Template,
 
+    /// The monitored resource to associate the logs with.
     pub resource: StackdriverResource,
+
+    /// The field of the log event from which to take the outgoing log’s `severity` field.
+    ///
+    /// The named field is removed from the log event if present, and must be either an integer
+    /// between 0 and 800 or a string containing one of the [severity level names][sev_names] (case
```

Comment:
👀 wth is a severity of 800?!

Reply:
Oh damn. 😂 Let me double check my copy/paste job...

Reply:
Huh, apparently it's [real](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#logseverity): each level maps to N*100, so 100, 200, 300.

I guess maybe they wanted to make it possible to allow ordering or something similar to priority levels for each severity level? 🤷🏻

Reply:
Yeah, that's just bonkers to me - too many choices!